src/swerex/deployment/config.py

```diff
@@ -95,7 +95,7 @@ class ModalDeploymentConfig(BaseModel):
     """Runtime timeout (default timeout for all runtime requests)
     """
 
-    deployment_timeout: float = 1800.0
+    deployment_timeout: float = 3600.0
```

Comment:
There's another timeout in `__init__` of ModalDeployment that would be good to change too I think